public/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/mstile-150x150.png"/>
+            <TileColor>#da532c</TileColor>
+        </tile>
+    </msapplication>
+</browserconfig>
```

Comment:
what is going on here?

Reply:
This was one of the imported files from creating our new favicon. I'm not entirely sure what it is doing exactly.

Reply:
ok that makes sense, thanks